feature-utils/poly-look/src/css/radius.css

```diff
@@ -0,0 +1,12 @@
+:root {
+    /* Border radius */
+      --pl-radius-1x: 4px;
+      --pl-radius-2x: calc(var(--pl-radius-1x) * 2);
+      --pl-radius-3x: calc(var(--pl-radius-1x) * 3);
+      --pl-radius-4x: calc(var(--pl-radius-1x) * 4);
+      --pl-radius-5x: calc(var(--pl-radius-1x) * 5);
+      --pl-radius-6x: calc(var(--pl-radius-1x) * 6);
+      --pl-radius-7x: calc(var(--pl-radius-1x) * 7);
+      --pl-radius-8x: calc(var(--pl-radius-1x) * 8);
```

Comment:
When we are going to need this amount of boilerplate, it's maybe about time we considered again SCSS or SASS or some CSS generator language; or we will have to write (and then later factorize) a lot...

Reply:
Let's keep in mind that our long-term aim is to have this information come from a SSOT. and so this file would already be automatically generated.